test/server.js

```diff
@@ -2198,4 +2198,31 @@ describe('server', function () {
       testForTransport('polling', done);
     });
   });
+
+  describe('response headers', function () {
+    function testForHeaders(headers, done) {
+      var engine = listen(function (port) {
+        var socket = new eioc.Socket('ws://localhost:%d'.s(port), {
+          extraHeaders: headers,
+          transports: ['polling']
+        });
+        engine.on('connection', function (conn) {
+          expect(conn.request.res._header).to.contain('X-XSS-Protection: 0');
```

Comment:
`res._header` doesn't look public field. Maybe we should use `getHeader()` instead?